fah_visualizer.py

```diff
@@ -13,14 +13,14 @@
     preload_top, preload_tops, save_trajs, save_generic, backup, load_trajs
```

Comment:
I was curious about your choice to import MSMBuilder vs. DeepTime. Is this going to install from the version of MSMBuilder that Xuhui Huang's lab is using? Or an older deprecated version? We should make sure we can consistently generate an environment where this will work